tests/integration/test_charm_fork_repo.py

```diff
@@ -75,7 +75,7 @@ async def test_dispatch_workflow_failure(
                 "Stopping execution of jobs due to repository setup is not compliant with policies"
                 in logs
             )
-            assert "Endpoint designed for testing that always fails" in logs
+            assert "The method is not allowed for the requested URL" in logs
```

Comment:
That doesn't seem right?

Reply:
The message from repo-policy-compliance is returning that. See:
https://github.com/canonical/github-runner-operator/actions/runs/7032231846/job/19139302478#step:14:167
After this change the integrations test are passing.

Since this is blocking other PR. Can we merge this PR first, then work on a fix on repo-policy-compliance and change this line back?

Reply:
The issue is that curl will automatically do a post if you send data, in this case no data is being sent. So `-X POST` should fix that issue

Reply:
So in this case curl issued a GET which resulted in a method now allowed error